integration_test/test_videoroom/assets/js/app.js

```diff
@@ -158,16 +131,22 @@ startMicOnlyButton.onclick = () => start("mic");
 startCameraOnlyButton.onclick = () => start("camera");
 startNoneButton.onclick = () => start("none");
 stopButton.onclick = stop;
-statsButton.onclick = refreshStats;
+statsButton.onclick = () => { refreshStats(remoteStreamsStats); }
 updatePeerMetadataButton.onclick = () => { room.updateMetadata() }
 updateTrackMetadataButton.onclick = () => { room.updateTrackMetadata() }
 peerMetadataButton.onclick = () => { putStats(room.peerMetadata) }
 trackMetadataButton.onclick = () => { putStats(room.trackMetadata) }
-lowSimulcastButton.onclick = () => { change(lowSimulcastButton, "l") }
-mediumSimulcastButton.onclick = () => { change(mediumSimulcastButton, "m") }
-highSimulcastButton.onclick = () => { change(highSimulcastButton, "h") }
-lowEncodingPeerButton.onclick = () => { room.changePeerSimulcastEncoding("l") }
-mediumEncodingPeerButton.onclick = () => { room.changePeerSimulcastEncoding("m") }
-highEncodingPeerButton.onclick = () => { room.changePeerSimulcastEncoding("h") }
-inboundSimulcastStatsButton.onclick = refreshSimulcastReceiverStats
-outboundSimulcastStatsButton.onclick = refreshSimulcastSenderStats
\ No newline at end of file
+localLowEncodingButton.onclick = () => { changeSimulcastEncoding(localLowEncodingButton, "l") }
+localMediumEncodingButton.onclick = () => { changeSimulcastEncoding(localMediumEncodingButton, "m") }
+localHighEncodingButton.onclick = () => { changeSimulcastEncoding(localHighEncodingButton, "h") }
+peerLowEncodingButton.onclick = () => { room.selectPeerSimulcastEncoding("l") }
+peerMediumEncodingButton.onclick = () => { room.selectPeerSimulcastEncoding("m") }
+peerHighEncodingButton.onclick = () => { room.selectPeerSimulcastEncoding("h") }
+inboundSimulcastStatsButton.onclick = () => {
+  refreshStats(async (connection) => {
+    let stats = await inboundSimulcastStreamStats(connection)
+    stats.callbackEncoding = room.getPeerEncoding()
```

Comment:
Would this make sens to do the following rename?
```suggestion
    stats.encoding = room.getPeerEncoding()
```